lms/templates/courseware/progress.html

```diff
@@ -124,7 +124,10 @@ <h2 class="title">${certificate_data.title | h}</h2>
                                         <span>${_("Verification Declined" )}</span>
                                     %elif requirement['status'] == 'satisfied':
                                         <span class="fa fa-check" aria-hidden="true"></span>
-                                        <span>${_("Completed by")} ${get_time_display(requirement['status_date'], DEFAULT_SHORT_DATE_FORMAT, settings.TIME_ZONE)}</span>
+                                        <%
+                                           data_string = _('Completed by {date}')
```

Comment:
No need to create a single-use temporary variable here that breaks the flow of the code.  Can you simply put the string within `data-string=` on the next line?